assume/markets/clearing_algorithms/nodal_pricing.py

```diff
@@ -10,7 +10,9 @@
 SOLVERS = ["glpk", "cbc", "gurobi", "cplex"]
 
 
-def nodal_pricing_pyomo(market_agent: MarketRole, market_products: list[MarketProduct]):
+def nodal_pricing_example(
```

Comment:
why should this be called "example"?
Its an implementation of nodal pricing using pyomo for the optimization, so I would prefer the previous name?

Reply:
because this example works only for a specific number of nodes (3), which is not a general case and should be only used as an example

Reply:
Then, we should fix this in a second issue instead of renaming it..? ;)

Reply:
yup agree